Framework/Kernel/inc/MantidKernel/Unit.h

```diff
@@ -632,13 +632,33 @@ class MANTID_KERNEL_DLL Degrees : public Empty {
 };
 
 //=================================================================================================
-
-/// Phi that has degrees as unit at "Phi" as title
+/// Phi that has degrees as unit and "Phi" as title
 class MANTID_KERNEL_DLL Phi : public Degrees {
   const std::string caption() const override { return "Phi"; }
   Unit *clone() const override { return new Phi(*this); }
 };
 
+//=================================================================================================
+/// Temperature in kelvin
+class MANTID_KERNEL_DLL TemperatureKelvin : public Empty {
```

Comment:
Why not call this simply `Temperature`? None of the other units mention the actual unit in their names, just the physical quantity's name.